src/features/unified-share-modal/__tests__/__snapshots__/UnifiedShareModalTitle.test.js.snap

```diff
@@ -16,7 +16,7 @@ exports[`features/unified-share-modal/HeaderTitle should render classifiction la
   <Classification
     className="bdl-UnifiedShareModalTitle-classification"
     definition="test"
-    messageStyle="tooltip"
+    messageStyle="bdl-Tooltip"
```

Comment:
Should non-classname related props be modified?

Reply:
Good catch. Will fix